test/test-sets/ref/aml_ccb_metrics.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"cbea_label_first_action":1,"cbea_label_not_first":3,"cbea_labeled_ex":4,"cbea_max_actions":2,"cbea_min_actions":2,"cbea_non_zero_cost":0,"cbea_predict_in_learn":0,"conf_idx_amls_0":0,"conf_idx_amls_1":7,"conf_idx_amls_2":6,"current_champ":0,"sfm_count_learn_example_with_shared":4,"test_county":4,"total_champ_switches":0,"total_learn_calls":4,"total_log_calls":0,"total_predict_calls":0,"upcnt_amls_0":0,"upcnt_amls_1":3,"upcnt_amls_2":3,"upcnt_sc_0":0,"upcnt_sc_1":3,"upcnt_sc_2":3,"bound_amls_0":0.0,"bound_amls_1":0.0,"bound_amls_2":0.0,"bound_sc_0":0.0,"bound_sc_1":0.0,"bound_sc_2":0.0,"cbea_avg_actions_per_event":2.0,"cbea_avg_feat_per_action":0.0,"cbea_avg_feat_per_event":0.0,"cbea_avg_ns_per_action":0.0,"cbea_avg_ns_per_event":0.0,"cbea_sum_cost":0.0,"cbea_sum_cost_baseline":0.0,"ips_amls_0":0.0,"ips_amls_1":0.0,"ips_amls_2":0.0,"ips_sc_0":0.0,"ips_sc_1":0.0,"ips_sc_2":0.0,"r_amls_0":0.0,"r_amls_1":-0.0,"r_amls_2":-0.0,"r_sc_0":0.0,"r_sc_1":-0.0,"r_sc_2":-0.0,"w_amls_0":0.0,"w_amls_1":0.0,"w_amls_2":0.0,"w_sc_0":0.0,"w_sc_1":0.0,"w_sc_2":0.0,"exclusionc_0":"{}","exclusionc_1":"{[\"[wild]\", \"[ccbid]\"]}","exclusionc_2":"{[\"U\", \"U\"]}","interactions_amls_0":"-q AA -q AS -q AU -q SS -q SU -q UU -q AA[ccbid] -q AA -q AS[ccbid] -q AS -q AU[ccbid] -q AU -q SS[ccbid] -q SS -q SU[ccbid] -q SU -q UU[ccbid] -q UU -q [wild][ccbid] ","interactions_amls_1":"-q AA -q AS -q AU -q SS -q SU -q UU -q AA[ccbid] -q AA -q AS[ccbid] -q AS -q AU[ccbid] -q AU -q SS[ccbid] -q SS -q SU[ccbid] -q SU -q UU[ccbid] -q UU -q [wild][ccbid] ","interactions_amls_2":"-q AA -q AS -q AU -q SS -q SU -q AA[ccbid] -q AA -q AS[ccbid] -q AS -q AU[ccbid] -q AU -q SS[ccbid] -q SS -q SU[ccbid] -q SU -q [wild][ccbid] "}
```

Comment:
"interactions_amls_1":"-q AA -q AS -q AU -q SS -q SU -q UU -q AA[ccbid] -q AA -q AS[ccbid] -q AS -q AU[ccbid] -q AU -q SS[ccbid] -q SS -q SU[ccbid] -q SU -q UU[ccbid] -q UU -q [wild][ccbid] "

in 

"exclusionc_1":"{[\"[wild]\", \"[ccbid]\"]}"

and its not being excluded, and it shouldn't be proposed as a valid exclusion